ext/pg_query/include/executor/executor.h

```diff
@@ -271,6 +271,12 @@ extern ProjectionInfo *ExecBuildProjectionInfo(List *targetList,
 											   TupleTableSlot *slot,
 											   PlanState *parent,
 											   TupleDesc inputDesc);
+extern ProjectionInfo *ExecBuildProjectionInfoExt(List *targetList,
+												  ExprContext *econtext,
+												  TupleTableSlot *slot,
+												  bool assignJunkEntries,
+												  PlanState *parent,
+												  TupleDesc inputDesc);
```

Comment:
Could you say a few words about the indentation strategy for this file? 

I can see there is some kind of alignment being used, is it to help a reader scan for groups of lines?

Reply:
That comes from the Postgres source, GitHub formats it a bit weird - it should be tab-based formatting, with 4 spaces considered one tab (i.e. the idea is that the E of "ExprContext" aligns with the L of "List")

Here is how it looks formatted locally for me:

<img width="599" alt="Screen Shot 2021-06-28 at 11 17 02 AM" src="https://user-images.githubusercontent.com/7227/123684509-6589dc80-d802-11eb-8736-94c3af6b1b07.png">

Reply:
Thanks, that makes sense!